src/main/java/org/jabref/model/pdf/FileAnnotation.java

```diff
@@ -106,7 +106,10 @@ private String parseContent(final String content) {
             return "";
         }
 
-        return content.trim();
+        // remove line breaks
+        String processedContent = content.replace("\r\n", " ").replace("\n", " ").replace("\r", " ");
```

Comment:
Can you please reuse the remove new lines formatter here, otherwise this is good to go.

Reply:
@tobiasdiez This does not work with our current conventions, i had it this way before...: Adhere to architecture tests do not use logic from within model

Reply:
Ok, thats again one of these cases where I don't like our strict architecture tests. One could argue that the Formatter can actually life in model as well, but I don't want to start a war here :-)

Reply:
me neither ;-)

Reply:
You can use the field formatter it in the view model of the file annotation instead of in the actual model (i.e. the actual model contains the text as it was in the pdf, but it is shown without line breaks).
https://github.com/JabRef/jabref/blob/efdf91a7d277a950204672c2c823d7e6dfe65305/src/main/java/org/jabref/gui/entryeditor/fileannotationtab/FileAnnotationViewModel.java#L19-L25